### DISCUSSION TITLE
Organise media content

### DISCUSSION THREAD

**Discussion by maximilianofried** [strapi/strapi#7830] [Legacy Topics]
Hello, I don't know if this is a thing yet. But it would be nice to organize content in different folders or something like that!

**Comment by darron1217** [comment]
This is definitely related to performance.

I've experienced storage read speed drop when I saved over 10k files in a directory.

After re-organizing files into `$year-$month-$day` formatted folder, the problem solved.

How about creating folders by upload date? (Just like wordpress)

**Comment by derrickmehaffy** [comment]
We have a feature request for folders on cloudinary: https://github.com/strapi/strapi/issues/2972 

But not for the local upload plugin (AWS-S3 already supports folders). Please feel free to submit a Pull Request thought :smiley:
